tests/test_mock_controller.py

```diff
@@ -65,90 +68,811 @@ async def test_missing_command_parameter(self):
+        await self.write("closeShutter:\n")
+        self.data = await self.read()
+        sau.assertReply("OK", self.data, Timeout=20)
+        await asyncio.sleep(1)
+        await self.write("status:\n")
+        self.data = await self.read()
+        sau.assertReply(
+            "ApSCS", self.data, status="Closed", positionActual=0.0, positionCmd=0.0,
+        )
+
+    async def test_stopShutter(self):
+        await self.write("openShutter:\n")
+        self.data = await self.read()
+        sau.assertReply("OK", self.data, Timeout=20)
+        await asyncio.sleep(1)
+        await self.write("stopShutter:\n")
+        self.data = await self.read()
+        sau.assertReply("OK", self.data, Timeout=20)
+        await asyncio.sleep(1)
+        await self.write("status:\n")
+        self.data = await self.read()
+        sau.assertReply(
+            "ApSCS", self.data, status="Stopped", positionActual=90.0, positionCmd=90.0,
+        )
 
     async def test_config(self):
-        config = {"AMCS": {"jmax": 0.1}}
+        config = {
+            "AMCS": {"jmax": 3.0, "amax": 0.75, "vmax": 1.5},
+            "LWSCS": {"jmax": 3.5, "amax": 0.875, "vmax": 1.75},
+        }
         await self.write(f"config:\n {config}\n")
         self.data = await self.read()
         sau.assertReply("OK", self.data, Timeout=20)
 
+    async def test_park(self):
+        await self.write("moveAz:\n azimuth: 10\n")
+        self.data = await self.read()
+        sau.assertReply("OK", self.data, Timeout=20)
+        await asyncio.sleep(0.2)
+        await self.write("park:\n")
+        self.data = await self.read()
+        sau.assertReply("OK", self.data, Timeout=20)
+        await asyncio.sleep(1)
+        await self.write("status:\n")
+        self.data = await self.read()
+        sau.assertReply(
+            "AMCS", self.data, status="Parked", positionActual=0.0, positionCmd=0.0
+        )
+
+    async def test_setTemperature(self):
+        await self.write("setTemperature:\n temperature: 10.0\n")
+        self.data = await self.read()
+        sau.assertReply("OK", self.data, Timeout=20)
+        await asyncio.sleep(1)
+        await self.write("status:\n")
+        self.data = await self.read()
+        self.log.info(f"data = {self.data}")
+        sau.assertReply(
+            "ThCS",
+            self.data,
+            status="Enabled",
+            data=[
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+                10.0,
+            ],
+        )
+
+    async def test_inflate(self):
+        await self.write("inflate:\n action: True\n")
+        self.data = await self.read()
+        sau.assertReply("OK", self.data, Timeout=20)
```

Comment:
The timeout value at some point will be calculated but for now I have replaced them with variables defined in mock_controller, since that determines the timeout value for now.